keyboards/keychron/k4/white/v2/rand.c

```diff
@@ -0,0 +1,31 @@
+// Small rand() implementation
```

Comment:
This doesn't belong in here, it should be on a higher level, for all keyboards..

Reply:
This is more on a system level, than for a specific keyboard, as it could be used by all keyboard vendors.. But I have no idea where we should put it.

Reply:
Ok, yeah I was thinking the same thing. It's useful for the more limited controllers. I was just copying it in to the K4 for now, until we can maybe get a standard one implemented somewhere in the project.